rubygem/lib/zeus/load_tracking.rb

```diff
@@ -1,8 +1,15 @@
 module Zeus
   class LoadTracking
     class << self
-      def features_loaded_by(&block)
-        old_features = all_features
+      def add_feature(file)
+        full_path = File.expand_path(file)
+        return unless File.exist?(full_path) && @feature_pipe
+        notify_features([full_path])
+      end
+
+      # Internal: This should only be called by Zeus code
+      def track_features_loaded_by
```

Comment:
I completely renamed this public method. I'm not too worried since I recently completely changed the return signature and no one complained. I marked it as internal so it shouldn't be a question in the future.